lambdas/data-migration1/src/index.ts

```diff
@@ -2,7 +2,6 @@ import { getKnexClient } from '@cumulus/db';
 import { MigrationSummary } from '@cumulus/types/migration';
 import Logger from '@cumulus/logger';
 
-import { migrateCollections } from './collections';
```

Comment:
I think at the point we need to do this sort of surgical removal, we're probably at the point we should just remove the tool entirely, rather than tear it down piecemeal.     I'm not sure it's *blocking* on this ticket, but it should be pretty trivial to remove.

Reply:
Did you mean to remove the whole data-migration1 entirely? Thanks

Reply:
Removed lambdas/data-migration1